### ISSUE TITLE
Errors in admonitions with code blocks

### ISSUE THREAD

**Issue by glenn-jocher** [KyleKing/mdformat-admon#17]
@KyleKing I'm trying to use this plugin with our docs at https://docs.ultralytics.com, and it creates incorrect output on adminitions that contain code blocks:

### Before running

```markdown
Ultralytics commands use the following syntax:

!!! Example

    === "CLI"

        ```bash
        yolo TASK MODE ARGS
        ```

    === "Python"

        ```python
        from ultralytics import YOLO

        # Load a YOLOv8 model from a pre-trained weights file
        model = YOLO('yolov8n.pt')

        # Run MODE mode using the custom arguments ARGS (guess TASK)
        model.MODE(ARGS)
        ```
```

### After running

```markdown
Ultralytics commands use the following syntax:

!!! Example
    === "CLI"

    ````
    ```bash
    yolo TASK MODE ARGS
    ```
    ````

    === "Python"

    ````
    ```python
    from ultralytics import YOLO

    # Load a YOLOv8 model from a pre-trained weights file
    model = YOLO('yolov8n.pt')

    # Run MODE mode using the custom arguments ARGS (guess TASK)
    model.MODE(ARGS)
    ```
    ````
```

The file that has this to reproduce the issue is at https://github.com/ultralytics/ultralytics/blob/main/docs/en/usage/cfg.md with `mdformat cfg.md --wrap no`

Could you take a look please and see if there is a fix that's possible? Thank you!

**Comment by KyleKing** [comment]
Looks like I haven't support Content Tabs yet! I will take a look this week to see what's involved

**Comment by KyleKing** [comment]
Actually this might be blocked. I knew I was way too familiar with this issue: https://github.com/KyleKing/mdformat-admon/issues/14

**Comment by glenn-jocher** [comment]
@KyleKing hey this is awesome! Thanks for the fast response.

**Comment by KyleKing** [comment]
Content Tabs should be now have full support as soon as I can release `mdformat-mkdocs`!

**Comment by fyliu** [comment]
> Content Tabs should be now have full support as soon as I can release `mdformat-mkdocs>=2.0.0`!

@KyleKing thank you for adding Content Tabs support! It's the next thing I would have asked for in mdformat-mkdocs since I use it in my docs as well.

**Comment by KyleKing** [comment]
I've been blocked releasing to PyPi until `mdformat-gfm` is released (https://github.com/hukkin/mdformat-gfm/issues/31), but I can provide a workaround and make a `v2.0.0rc1` release in the interim

**Comment by KyleKing** [comment]
`v2.0.0rc2` is now available as long as you install `mdformat-gfm` separately: <https://pypi.org/project/mdformat_mkdocs/2.0.0rc2>

**TLDR**: setup pre-commit like this:

```yaml
repos:
  - repo: https://github.com/executablebooks/mdformat
    rev: 0.7.17
    hooks:
      - id: mdformat
        additional_dependencies:
          - mdformat-mkdocs[recommended]>=2.0.0rc2
          - mdformat-gfm@git+https://github.com/hukkin/mdformat-gfm.git
```
